source/vocab/things.ttl

```diff
@@ -524,8 +523,8 @@
     rdfs:label "Projected Image"@en, "Projicerad bild"@sv;
     skos:exactMatch rdamedia:1005 ; # "projected" ("projected graphic", "motion picture")
     :inCollection marc:typeOfRecord;
-    # TODO: Work should not be necessary to set as subClassOf since Visual is subClass of Work. Should be removed with LXL-2735
```

Comment:
For a clean revert, we may want to remove the workaround in a separate commit after LXL-2735 has landed. @sarwik any opinions?